cmd/dagger/main.go

```diff
@@ -253,6 +254,10 @@ func main() {
 		rootCmd.SetOut(stdout)
 		rootCmd.SetErr(stderr)
 
+		if url, ok := telemetry.URLForTrace(ctx); ok {
+			slog.Info("Connected to cloud", "url", url)
+		}
```

Comment:
So this works :tada: At this point we have a root span, so we have a trace ID so can get the URL for the telemetry to be viewed at - but I'm not sure, at this point is this page guaranteed to exist? At what point can we guarantee this (and defer printing till then ideally)?